MSAL/src/public/MSALDefinitions.h

```diff
@@ -154,6 +154,19 @@ typedef NS_ENUM(NSUInteger, MSALDeviceMode)
     MSALDeviceModeShared
 };
 
+/**
+ Device mode configured by the administrator
+ */
+typedef NS_ENUM(NSUInteger, MSALSSOExtensionMode)
```

Comment:
I don't think we need to expose device mode publicly in MSAL... is there any value from it to outside developers?